dom.go

```diff
@@ -181,7 +185,7 @@ func (h *HTML) reconcile(prev *HTML) {
 			h.node.Set(name, value)
 		}
 	}
-	if !h.new {
+	if h.node == prev.node {
```

Comment:
For readability sake I'd like to declare this as `new := h.node != prev.node` and continue using `!new` in the if statement. I'll make this change before merging.

Reply:
I'm online currently, happy to edit this in if you'd like.

Reply:
If you're still online, please do :)

Reply:
Actually, just cherry-pick my commit onto your branch if you don't mind: https://github.com/gopherjs/vecty/commit/059f8593ecdd7c13ffee79da47ebdaad6448e244

Reply:
Sorry, please disregard my last two comments and see my comment below instead.